requirements_linux.txt

```diff
@@ -0,0 +1,15 @@
+geopandas
```

Comment:
I renamed this file to requirements_linux - could you check if test runs properly before merging?

Reply:
At least it seems to me as if it worked :) 

Also I am surprised that tests on the linux virtual machine ran significantly faster than the super linter. But maybe this will change, the more tests we write :)

![grafik](https://user-images.githubusercontent.com/53005443/163492546-57c59d4d-fb4e-4ba9-be8b-841066287eae.png)

Reply:
Yes everything looks good to me~

Reply:
Yes, the super-linter runs several types of linters. We could also consider incorporating the listing directly in the testing workflow. This would probably speed things up :) But for now it works fine~

Reply:
@randy-seng ok from my perspective you can merge the PR. I assigned you :)

Dunno if @frunika wants to have a look?